systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -47,8 +47,11 @@
     long GLOBAL_TRACING_POLL = Duration.ofSeconds(30).toMillis();
     long GLOBAL_TRACING_TIMEOUT =  Duration.ofMinutes(7).toMillis();
 
+    long GLOBAL_CRUISE_CONTROL_TIMEOUT = Duration.ofSeconds(10).toMillis();
```

Comment:
You should be careful with this timeout. The REST API default timeout is 10seconds so you may timeout the tests before you get a response of any kind from the server.

Reply:
So do you suggesting to make this `GLOBAL_CRUISE_CONTROL_TIMEOUT` little bit bigger? Would be suitable 30s ?

Reply:
Well anything larger that 10secs, to be safe 20 should do.